src/celengine/body.cpp

```diff
@@ -544,9 +544,9 @@ UniversalCoord Body::getPosition(double tdb) const
 {
     Vector3d position = Vector3d::Zero();
 
-    const TimelinePhase* phase = timeline->findPhase(tdb);
+    auto phase = timeline->findPhase(tdb);
```

Comment:
why did you remove `const` here

Reply:
Probably because returned reference is const already.

Reply:
Probably because returned reference is const already.